pkgs/applications/audio/reaper/default.nix

```diff
@@ -12,6 +12,7 @@
 , which
 
 , jackSupport ? true, libjack2
+, pipewireJackSupport ? false, pipewire
```

Comment:
There doesn't seem to be an equivalent of `config.pulseaudio` for enabling pipewire, but maybe I'm missing something?